src/components/results/generic/index.js

```diff
@@ -0,0 +1,48 @@
+import {useState, useEffect} from "react";
+import useGraphql from "lib/hooks/use-graphql";
+import useHttp from "lib/hooks/use-http";
+import useResults from "lib/hooks/use-results";
+import Breakdown from "./breakdown";
+import Header from "./header";
+import ResultActions from "./result_actions";
+import Score from "./score";
+import style from "./style.scss";
+
+export default function Generic() {
+  const [result, setResult] = useState(null);
+  const assessment = useResults({surveyType: "generic"});
+  const profile = result ? result.profile : {};
+  const assessmentResult = result ? result.assessment : {};
+
+  const graphQL = useGraphql();
+  const http = useHttp();
+
+  useEffect(() => {
+    if(assessment === null) return;
+    const query = graphQL.generic.result;
+    const variables = {assessmentID: assessment.id};
+
+    http.post(graphQL.generic.path, {query, variables}).then(({data, errors}) => {
+      if(!errors && data.genericAssessmentResult) {
+        setResult(data.genericAssessmentResult);
+      } else {
+        console.warn(errors || data); // eslint-disable-line no-console
+      }
+    });
+  }, [assessment]);
+
+  return (
+    <div>
+      {result && (
+        <div className={style.container}>
```

Comment:
Sorry you put all the work into building them out, but the widgets just needs the Score and the Breakdown parts. The header and actions are just what it looks like within Dash or Olivia

Reply:
Make sure you remove the extra css too please